api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -4896,3 +4931,89 @@ message OktaAssignmentResult {
     (gogoproto.jsontag) = ""
   ];
 }
+
+// AccessListResource is emitted when an access list is created, updated, or deleted.
```

Comment:
Don't we typically create dedicated events for create, update, delete?

(I see `AppCreate`, `DatabaseUpdate`, `MFADeviceDelete` etc.)

Any particular reason to break from that pattern?

Reply:
It seems needlessly duplicative TBH. I could create three different `AccessListCreate/Update/Delete` events, but they'll all be the same.

I can adjust this if you feel strongly, but after wading through this a bit it seems like most modification operations really have the same metadata save for the event name/code.

Reply:
I don't feel super strongly, but whenever there is an inconsistency people ask why and wonder if there's something they are missing.

IMO, whether or not the original decision was the correct one, there is value in sticking to the same conventions rather than coming up with a new approach every time. You _did_ do separate events for the SAML IdP service provider only 6 months ago..